main.go

```diff
@@ -183,8 +186,8 @@ func main() {
 
 	router := route.New()
 
-	RegisterWeb(router)
-	api.Register(router.WithPrefix("/api"))
+	RegisterWeb(router.WithPrefix(amURL.Path))
+	api.Register(router.WithPrefix(amURL.Path + "/api"))
```

Comment:
Note that this is not an issue here though, since we're dealing with paths only. Both `path.Join` or `+` would work fine, just that I think `+` is what we usually use in Prometheus in this situation.